### ISSUE TITLE
eval_beamsearch_ngram.py not working

### ISSUE THREAD

**Issue by phypan11** [NVIDIA/NeMo#3077]
**Describe the bug**

eval_beamsearch_ngram.py not working because logprobs argument is not defined in rnnt_model.transcribe
**Steps/Code to reproduce bug**

run eval_beamsearch_ngram.py as described in  https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/asr_language_modeling.html#evaluate-by-beam-search-decoding-and-n-gram-lm

**Expected behavior**

If I remove logprobs argument, it produces string and makes an error because it is fed to softmax.

**Environment overview (please complete the following information)**

 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider - AWS, Azure, GCP, Collab)]
 - Method of NeMo install: [pip install or from source]. Please specify exact commands you used to install.
 - If method of install is [Docker], provide `docker pull` & `docker run` commands used

**Environment details**

If NVIDIA docker image is used you don't need to specify these.
Otherwise, please provide:
- OS version
- PyTorch version
- Python version

**Additional context**

Example: GPU model

**Comment by VahidooX** [comment]
eval_beamsearch_ngram.py script does not support Transducer models currently. It is on our roadmap to add this support. It needs some updates to other modules and scripts and it is not that simple. Also, keep in mind that the improvements you may get from language models for Transducer models are very limited comparing to CTC models.

**Comment by manjuke** [comment]
Hi @VahidooX , I still get same error, if I try to use eval_beamsearch_ngram.py script for decoding rnnt o/p. Is this feature implemented already?. Is there any workaround to do this. Thanks

**Comment by VahidooX** [comment]
We need some more time to add such a feature with reasonable inference speed. It is not trivial to make it as fast as the beam search decoding for CTC models. Currently if you are looking to use LM, a CTC model with LM would likely to give better and faster results comparing to greedy RNNT model.

I did added this feature long time ago in an experimental branch but we did not add it to the main branch. You may play with this one:
https://github.com/NVIDIA/NeMo/issues/3412#issuecomment-1051484264
